bukkit/src/main/java/de/maxhenkel/voicechat/config/ServerConfig.java

```diff
@@ -30,79 +30,79 @@ public ServerConfig(ConfigBuilder builder) {
+                        "The IP address to bind the voice chat server to",
+                        "Leave blank to use the 'server-ip' property from server.properties",
                         "To bind to the wildcard address, use '*'"
                 );
         voiceChatDistance = builder
                 .doubleEntry("max_voice_distance", 48D, 1D, 1_000_000D,
-                        "The distance to where the voice can be heard"
+                        "The distance to which the voice can be heard"
                 );
         crouchDistanceMultiplier = builder
                 .doubleEntry("crouch_distance_multiplier", 1D, 0.01D, 1D,
-                        "The multiplier the voice distance will be reduced by when sneaking"
+                        "The multiplier of the voice distance when crouching"
                 );
         whisperDistanceMultiplier = builder
                 .doubleEntry("whisper_distance_multiplier", 0.5D, 0.01D, 1D,
-                        "The multiplier the voice distance will be reduced by when whispering"
+                        "The multiplier of the voice distance when whispering"
                 );
         voiceChatCodec = builder
                 .enumEntry("codec", Codec.VOIP,
-                        "The opus codec"
+                        "The Opus codec",
+                        "Valid values are 'VOIP', 'AUDIO', and 'RESTRICTED_LOWDELAY'"
                 );
         voiceChatMtuSize = builder
                 .integerEntry("mtu_size", 1024, 256, 10000,
-                        "The maximum size in bytes in a voice packet",
-                        "Set this to a lower value if your voice packets don't arrive"
+                        "The maximum size that voice packets are allowed to have (in bytes)"
                 );
         keepAlive = builder
                 .integerEntry("keep_alive", 1000, 1000, Integer.MAX_VALUE,
-                        "The frequency in which keep alive packets are sent",
+                        "The frequency at which keep-alive packets are sent (in milliseconds)",
                         "Setting this to a higher value may result in timeouts"
                 );
         groupsEnabled = builder
                 .booleanEntry("enable_groups", true,
-                        "If group chats are allowed"
+                        "Controls whether group chats are allowed"
                 );
         voiceHost = builder
                 .stringEntry("voice_host", "",
-                        "The host name that clients should use to connect to the voice chat",
+                        "The hostname that clients should use to connect to the voice chat",
                         "This may also include a port, e.g. 'example.com:24454'",
-                        "Don't change this value if you don't know what you are doing"
+                        "Do NOT change this value if you don't know what you're doing"
```

Comment:
In the translation you changed you're to you are. Thats just arbitrary.

Reply:
The length doesn't really matter, the "NOT" is good though.

Reply:
Should I get rid of the contraction and change it back to "you are", or is it fine to leave it as is?

Reply:
We should keep it consistent with the other config comments.

Reply:
The comments in all the configuration files contain both full forms and contractions, and that was the case even before I started making any changes. Therefore, I think it's fine to leave it as is for now, and any further changes of this kind can be left for future commits or pull requests.